Cargo.toml

```diff
@@ -13,6 +13,7 @@ members = [
     "mapper/cumulocity/c8y_smartrest",
     "mapper/tedge_mapper",
     "mapper/thin_edge_json",
+    "sm/download",
```

Comment:
To be honest it makes no sense to me to have this as a separate crate, just added it because it was specified as requirement, I would rather have it a module for `plugin_sm` crate at the moment as there is nothing else than agent using it...